scala-core-fp/src/main/scala/com/baeldung/scala/functor/Functor.scala

```diff
@@ -0,0 +1,71 @@
+package com.baeldung.scala.functor
```

Comment:
I don't think the files in this package belong in the PR for this article

Reply:
fixed